YAXLib/Pooling/ObjectPools/PoolPolicy.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors.
+// Licensed under the MIT license.
+
+#nullable enable
+using System;
+
+namespace YAXLib.Pooling.ObjectPools;
+
+/// <summary>
+/// Defines the configuration of how an <see cref="IObjectPool{T}"/> works.
+/// </summary>
+/// <typeparam name="T">The type of item being pooled.</typeparam>
+internal class PoolPolicy<T> where T : class
+{
+    private uint _maximumPoolSize = 10000;
```

Comment:
A lot in terms of the current extent of using pooling? Too much?

Reply:
What do you mean with this comment?

Reply:
There's only the `YAXSerializerPool` right now, and for that 50 would be enough, as it goes together with max recursion.

Reply:
Ah, no: `StringbuilderPool` is also in the game, but also with rather low usage.